### DISCUSSION TITLE
Error when trying to start the bot

### DISCUSSION THREAD

**Discussion by John-Harms** [jef/streetmerchant#1024] [Help]
Whenever I run the start command, I get the error below.  The street merchant folder is in the C:\Users\[UserName]> Directory, but the package.json isn't.  I don't really know much about how to solve this as I'm kinda dumb, so help would be appreciated!

Microsoft Windows [Version 10.0.18363.1198]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\Users\[UserName]>npm run start
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path C:\Users\[UserName\package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\[UserName]\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\[UserName]\AppData\Roaming\npm-cache\_logs\2020-11-30T03_44_37_523Z-debug.log

C:\Users\[UserName]>

This is what I got when downloading the street merchant:

Chromium (818858) downloaded to C:\Users\[UserName]\streetmerchant\node_modules\puppeteer\.local-chromium\win64-818858

> husky@4.3.0 postinstall C:\Users\[UserName]\streetmerchant\node_modules\husky
> opencollective-postinstall || exit 0

Thank you for using husky!
If you rely on this package, please consider supporting our open collective:
> https://opencollective.com/husky/donate

> nodemon@2.0.6 postinstall C:\Users\[UserName]\streetmerchant\node_modules\nodemon
> node bin/postinstall || exit 0

> twitch@4.3.2 postinstall C:\Users\[UserName]\streetmerchant\node_modules\twitch
> node ./scripts/npm/postinstall.js

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 1379 packages from 1434 contributors and audited 1383 packages in 219.074s

108 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

**Comment by jef** [comment]
I believe you're in the wrong directory. Take a look at https://github.com/jef/streetmerchant/wiki/Installation:-Windows.

**Comment by DeLanoyM99** [comment]
I think its because you typed npm install before you cd into the directory
